test/Ocelot.UnitTests/Requester/MessageInvokerPoolTests.cs

```diff
@@ -73,7 +73,7 @@ public void If_two_delegating_handlers_are_defined_then_these_should_be_call_in_
             .And(x => GivenADownstreamRoute("/super-test"))
             .And(x => GivenAMessageInvokerPool())
             .And(x => GivenARequest())
-            .When(x => WhenICallTheClient("http://www.bbc.co.uk"))
+            .When(x => WhenICallTheClient("https://www.google.com"))
```

Comment:
Why?

Reply:
Is DNS traffic banned for clients from Iran? 🤣
I'm sorry to be having this trouble. 🙊